CMakeLists.txt

```diff
@@ -154,19 +154,16 @@ add_to_config(_buildInfo COMPILE_HIP_ATP_MARKER)
 # Detect profiling API
 ################
 if(USE_PROF_API EQUAL 1)
-    if(NOT DEFINED PROF_API_HEADER_PATH)
-        find_package(PkgConfig)
-        pkg_check_modules(ROCTRACERPROTO QUIET roctracer-proto)
-        if(ROCTRACERPROTO_FOUND EQUAL 1)
-            pkg_get_variable(PROF_API_HEADER_PATH roctracer-proto prefix)
-        else()
-            set(PROF_API_HEADER_PATH /opt/rocm/roctracer/include/ext)
-        endif()
-    endif()
-    find_path(PROF_API_HEADER_DIR NAMES prof_protocol.h PATHS ${PROF_API_HEADER_PATH} NO_DEFAULT_PATH)
+    find_path(PROF_API_HEADER_DIR prof_protocol.h
+        HINTS
+            ${PROF_API_HEADER_PATH}
+        PATHS
+            /opt/rocm/roctracer
+        PATH_SUFFIXES
+            include/ext
+    )
     if(NOT PROF_API_HEADER_DIR)
-        MESSAGE(STATUS "PROF_API_HEADER_PATH = ${PROF_API_HEADER_PATH}")
-        MESSAGE(WARNING "Profiling API header not found. Disabling roctracer integration. Use -DPROF_API_HEADER_PATH=<path to prof_protocol.h header>")
+        MESSAGE(FATAL_ERROR "Profiling API header not found. Disabling roctracer integration. Use -DPROF_API_HEADER_PATH=<path to prof_protocol.h header>")
```

Comment:
I don't think this should be a fatal error. We can still build HIP runtime albeit without roctracer integration. So it is sufficient to give a warning that the profiling API header is not found and roctracer integration is disabled. This is important for CI until 2.1.x is released. Also we still have folks building tip of HIP on 2.0.x.

Reply:
'pkg-config' is removed with find_path.
I just worry that if the roctracer-proto header will be installed in wrong place or not installed by mistake it will be easy to check with the error.
Could they build 2.0.x version without USE_PROF_API enabled?